src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,136 @@
+import { useCallback } from 'react';
+import cn from 'classnames';
+import { useParams, useSearchParams } from 'react-router-dom';
+import { SearchParams } from '../../utils/searchHelper';
+import { SearchLink } from '../SearchLink';
+import { PersonLink } from '../PersonLink';
+import { Person } from '../../types';
+import { proceedPeople } from '../../utils/proceedPeople';
+
+type Props = {
+  people: Person[],
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { humanId } = useParams();
+  const [searchParams] = useSearchParams();
+
+  const getSortingParams = useCallback((param: string): SearchParams => {
+    if (searchParams.get('sort') === param
+      && searchParams.get('order') === 'desc') {
+      return { sort: null, order: null };
+    }
+
+    if (searchParams.get('sort') === param) {
+      return { sort: param, order: 'desc' };
+    }
```

Comment:
Use [`switch case`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) in cases where you constantly compare one value with others

Reply:
Your suggestion makes code harder to understand
```
switch (searchParams.get('sort')) {
      case param:
        if (searchParams.get('order') === 'desc') {
          return { sort: null, order: null };
        }

        return { sort: param, order: 'desc' };
      default: return { sort: param, order: null };
    }
```
I`d preffer to leave my code as it appears